### ISSUE TITLE
ride can not handle argument variables with spaces included 

### ISSUE THREAD

**Issue by robotframework-ride** [robotframework/RIDE#1293]
> _Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1255) by immo.wetzel on 3 Jun 2013_

As described in
<a href="https://groups.google.com/forum/?fromgroups#!topic/robotframework-users/JqyHSyd6vc4" rel="nofollow">https://groups.google.com/forum/?fromgroups#!topic/robotframework-users/JqyHSyd6vc4</a>
I used double quotes to save the spaces on the command line 

pybot --variable "myVar:D:/my test/file.var" --loglevel DEBUG "D:\my test/myTest.txt"

But if I copy this line to the arguments part of Ride the log out put shows a different behavior.

command: pybot.bat --variable '"myVar:D:/my test/file.var:D:/my' 'test/file.var"' --loglevel DEBUG ....

the variable is completly destroyed.

I guess Ride should behave like the command line version...

Used Ride version: 1.1
on Windows XP SP3

**Comment by robotframework-ride** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=1255#c1) by immo.wetzel on 3 Jun 2013*


copy and paste error

Ride shows:
command: pybot.bat --variable '"myVar:D:/my' 'test/file.var"' --loglevel DEBUG ....

**Comment by HelioGuilherme66** [comment]
It is 2021 and now I found this bug. Really annoying :(.
I passed this in arguments:  `--variable "external_var:This is external var definition"` or  `--variable "external_var:This is external var definition"`
`robot` in command line (shell) works fine.
On RIDE we get red color on arguments text box, and tool tip shows: `Unknown option(s): ['is', 'external', 'var', 'definition"']`
The processed argument file is: /tmp/RIDE9qvbvis8.d/argfile.txt 
```
--variable
external_var:"This
is
external
var
definition"
```
This is example test case:
```robot
*** Variables ***
${abc}            This is a String with ${external_var}
${def}            This is another string with ${CURDIR}

*** Test Cases ***
Log Variable
    Log    ${def}
    Log    ${abc}
```
Info:
```
20210810 22:48:13.509 [INFO]: Found Robot Framework version 4.1 from /home/helio/.local/lib/python3.9/site-packages/robot.

20210810 22:48:13.510 [INFO]: Started RIDE v2.0b2.dev3 using python version 3.9.6 with wx version 4.1.1 in linux.
```

**Comment by HelioGuilherme66** [comment]
Fixed by #2453
